src/js/health-records/reducers/refresh.js

```diff
@@ -0,0 +1,22 @@
+import _ from 'lodash';
+
+const initialState = {
+  statuses: {
+    OK: [],
+    ERROR: [],
+  },
+};
+
+export default function refresh(state = initialState, action) {
+  switch (action.type) {
+    case 'REFRESH_POLL_SUCCESS': {
+      // returns group in form {'OK': [], 'Error': []}
+      const statuses = _.groupBy(action.data, 'attributes.status');
```

Comment:
I left this as `groupBy` in case we wanted to show which refreshes failed out. It is only possible to refresh once per day, so if one of the updates fails, there is no way to refresh that data again until the next day.

Given this limitation, we can potentially add improved messaging down the line that specifies which part of the update failed, and let the veteran make a choice on whether to proceed.